_layouts/provider.html

```diff
@@ -17,54 +17,22 @@
 
 <body>
     <div class="content">
-        <a href="./">&lt;Back to Overview</a><br>
-        <h1>{{ page.name }} <small style="color: grey">Provider Info</small>
-        </h1>
-        <hr>
-        <div class="section">
-            <b>domains</b>: {{ page.domains }}
-            <!-- Can be array (todo handle that case) -->
-            <br>
-            <b>Authentication</b>: {{ page.credentials }}
-            <br>
-            <b>STATUS</b>:
-            <span class="status-minimal status-{{ page.status.state | downcase }}">
-                {% case page.status.state %}
-                {% when 'OK' %}
-                OK
-                {% when 'PREP' %}
-                PREPARATION NEEDED
-                {% when 'BROKEN' %}
-                NOT WORKING
-                {% endcase %}
-                on {{ page.status.date }}
-            </span>
-            <br>
-            <b>Limits</b>:
-            {%if page.limits %}
-                {{ page.limits }}
-            {%else%}
-                <i>Unknown</i>
-            {%endif%}
-            <br>
-            <b>Account Registration</b>: 
-            {%if page.registration %}
-                {{ page.registration }}
-            {%else%}
-                <i>Unknown</i>
-            {%endif%}
-            {%if page.website%}
-                <br>
-                <b>Website</b>: <a href="{{ page.website }}">{{ page.website }}</a>
-            {%endif%}
-        </div>
-        <hr>
-        <div class="section">
-            {{ content }}
+      <img src='/assets/provider-db-logo.svg' class='provider-db-logo' alt=''/>
```

Comment:
On the staging instance, the image path seems broken: https://testrun.org/provider/3fd1f6b32ecf43c7834594f7b058744e178a809f/_site/aktivix-org

`'./assets[...]` seems to do the trick :)

Reply:
Good catch, thanks! Fixed in 8ed07e8.